app/assets/javascripts/filter_projects.js

```diff
@@ -108,13 +108,27 @@ App.StimulusApp.register('filter-projects', class extends Stimulus.Controller {
     return data
   }
 
+  rawFormFiltersInput() {
+    const inputs = {}
+    $(this.element).find(':input').each((i, el) => {
+      // Hack to convert nasty rails param format into a functional JSON object
+      if(el.name.startsWith('filter[')) {
+        let name = el.name.replace('filter[', '').replace('][]', '').replace(/\]$/, '')
+        let val = $(el).val();
+        inputs[name] = val;
+      }
```

Comment:
This is functional but it makes me a little uncomfortable that it will break if we ever introduce another level of nesting in any of our report filters.
For example:
```js
let key = 'filter[client][name]';
key.replace('filter[', '').replace('][]', '').replace(/\]$/, '');
=> 'client][name'
```

We could support deeply nested objects generically but that's more some complexity than we might need. Perhaps we can just alert to sentry if we encounter a field we can't process? 

___Note:___ it's not clear to me that we actually have client side js sentry in the WH. I'm assuming we do and I'm just missing it

```suggestion
      // not tested
      if(el.name.startsWith('filter[')) {
        const matches = el.name.match(/^filter\[([^\]]+)\](\[\])?$/); // matches one level of nesting
        const fieldName = matches ? matches[1] : null;

        if (fieldName) {
          inputs[fieldName] = $(el).val();
        } else {
           // deeply nested fields are not supported
           Sentry.captureMessage(`field format not supported: "${el.name}"`);
        }
      }
```

Reply:
Pretty sure we _don't_ have client side sentry in the warehouse.

Reply:
Got it, thanks. Feels like we should have that [tracking that here](https://github.com/open-path/Green-River/issues/6837)